src/main/java/seedu/address/logic/commands/DeleteInterestCommand.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.interest.Interest;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a list of interests to the specified person.
```

Comment:
Deletes

Reply:
Sorry about that, seemed to miss out some